parametric_stellarator.py

```diff
@@ -206,6 +208,16 @@ def exports(export, components, magnets, logger):
         for name, comp in components.items():
             cq.exporters.export(comp['solid'], name + '.step')
     
+    if magnets['meshing']:
+        # Get current working directory
+        cwd = os.getcwd()      
+        # Exodus export
+        exo_path = f'{cwd}/coil_mesh.exo'
+        cubit.cmd(f'export mesh "{exo_path}"')
+        # h5m conversion
+        h5m_path = f'{cwd}/coil_mesh.h5m'
+        h5m_conversion = subprocess.run(f'mbconvert {exo_path} {h5m_path}', shell = True)
```

Comment:
I think we can do this with pymoab commands by just reading and writing a file in different formats, possibly in a new MOAB instance but maybe just into a new meshset?  It will be lighter weight than launching a subprocess.

Reply:
I looked in the MOAB bitbucket for a pymoab version of mbconvert, but could not find anything. Is there a specific function that you have in mind to replace the subprocess command?

Reply:
Something like:
```
myFileset = pymoab.EntitySet()
myMoab.read_file('file.exo', fileset = myFileset)
myMoab.write_file('file.h5m', fileset = myFileset)
```